applications/MeshingApplication/custom_utilities/metrics_math_utils.h

```diff
@@ -115,46 +113,40 @@ class MetricsMathUtils
         return Tensor;
     }
     
-    /***********************************************************************************/
-    /***********************************************************************************/
-    
     /**
      * It computes the intersection between two metrics 
-     * @param Metric1: The first metric
-     * @param Metric2: The second metric
+     * @param Metric1 The first metric
+     * @param Metric2 The second metric
      */
         
     static inline Vector IntersectMetrics(
-        const Vector Metric1,
-        const Vector Metric2
+        const Vector& Metric1,
```

Comment:
@rubenzorrilla I added some & additionally to use the reference instead of a copy